assets/javascripts/discourse/initializers/chat-setup.js

```diff
@@ -129,6 +132,82 @@ export default {
           }
         });
       });
+      api.addSidebarSection((BaseSectionHeader, BaseSectionLink) => {
+        return class extends BaseSectionHeader {
+          @tracked sectionLinks = A([]);
+
+          constructor() {
+            super(...arguments);
+            this.chatService = container.lookup("service:chat");
```

Comment:
If this is a service, you can do something like `@service chat` instead of having to look it up.